core/lib/FrameEnvironment.ts

```diff
@@ -241,6 +241,7 @@ export default class FrameEnvironment {
       result: entry.result,
       runStartDate,
       endDate,
+      run: this.session.resumeCounter,
```

Comment:
I find this `run` key to be very vague and confusing.

Reply:
Is it the name "run" or just that it doesn't have "id" on it or "counter"? Like, would "runId" make it clearer?